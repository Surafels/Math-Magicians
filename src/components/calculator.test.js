import React from 'react';
import renderer from 'react-test-renderer';
import Calculator from './calculator';

test('should render correctly', () => {
  const tree = renderer.create(<Calculator />).toJSON();
  expect(tree).toMatchSnapshot();
});
