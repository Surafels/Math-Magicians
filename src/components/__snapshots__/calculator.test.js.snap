// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<section>
  <p>
    Lets do some math
  </p>
  <div
    className="Container"
  >
    <input
      className="output grid-element"
      onClick={[Function]}
      type="button"
      value="0"
    />
    <input
      className="grid-element"
      onClick={[Function]}
      type="button"
      value="AC"
    />
    <input
      className="grid-element"
      onClick={[Function]}
      type="button"
      value="+/-"
    />
    <input
      className="grid-element"
      onClick={[Function]}
      type="button"
      value="%"
    />
    <input
      className="grid-element operator"
      onClick={[Function]}
      type="button"
      value="รท"
    />
    <input
      className="grid-element"
      onClick={[Function]}
      type="button"
      value="7"
    />
    <input
      className="grid-element"
      onClick={[Function]}
      type="button"
      value="8"
    />
    <input
      className="grid-element"
      onClick={[Function]}
      type="button"
      value="9"
    />
    <input
      className="grid-element operator"
      onClick={[Function]}
      type="button"
      value="x"
    />
    <input
      className="grid-element"
      onClick={[Function]}
      type="button"
      value="4"
    />
    <input
      className="grid-element"
      onClick={[Function]}
      type="button"
      value="5"
    />
    <input
      className="grid-element"
      onClick={[Function]}
      type="button"
      value="6"
    />
    <input
      className="grid-element operator"
      onClick={[Function]}
      type="button"
      value="-"
    />
    <input
      className="grid-element"
      onClick={[Function]}
      type="button"
      value="1"
    />
    <input
      className="grid-element"
      onClick={[Function]}
      type="button"
      value="2"
    />
    <input
      className="grid-element"
      onClick={[Function]}
      type="button"
      value="3"
    />
    <input
      className="grid-element operator"
      onClick={[Function]}
      type="button"
      value="+"
    />
    <input
      className="grid-element zero-button"
      onClick={[Function]}
      type="button"
      value="0"
    />
    <input
      className="grid-element"
      onClick={[Function]}
      type="button"
      value="."
    />
    <input
      className="grid-element equal-sign"
      onClick={[Function]}
      type="button"
      value="="
    />
  </div>
</section>
`;
